.github/workflows/playwright.yml

```diff
@@ -0,0 +1,50 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test_setup:
+    name: Test setup
+    runs-on: macos-latest
```

Comment:
Why use macos-latest here and macos-12 later?

Reply:
Thanks for catching that. I realized the second job's OS is important, changed it, and forgot to update the OS of the first job 😆 . But the `test_setup`'s OS would not affect the test or anything since it is only the placeholder for the next actual test job. I'll update the yml on my next PR.